src/blocks/ProductList/product/Component.tsx

```diff
@@ -1,32 +1,136 @@
 import React from 'react'
 
-import type { EuProduct } from '@/payload-types'
+import type { Category, Country, EuProduct, ReplacedProduct, Subcategory } from '@/payload-types'
 
 import { Media } from '@/components/Media'
+import { Card, CardContent, CardDescription, CardFooter, CardHeader } from '@/components/ui/card';
+import { H2, H4 } from '@/components/ui/typography';
+import { Button } from '@/components/ui/button';
+import { Link } from '@payloadcms/ui';
 
 interface ProductCardProps {
   product: EuProduct;
 }
 
 export const ProductCard: React.FC<ProductCardProps> = async ({ product }) => {
```

Comment:
![img_o9jWzk32@2x](https://github.com/user-attachments/assets/896eb0c9-28d7-48dd-a528-7268a2865bb6)
The Cards are not very responsive. The layout breaks when resizing the screen.